atlasdb-config/src/main/java/com/palantir/atlasdb/config/AtlasDbConfig.java

```diff
@@ -131,7 +131,10 @@ public int getBackgroundScrubBatchSize() {
      * have been overwritten or deleted. This differs from scrubbing
      * because it is an untargeted cleaning process that scans all data
      * looking for cells to delete.
+     * @deprecated Use {@link AtlasDbRuntimeConfig#enableSweep} to make this value
+     * live-reloadable.
```

Comment:
Given that the configs are created separately - not sure if this is possible. Perhaps some validation (and logging a warning if it fails) in `TransactionManagers.create`?